portfolio/src/main/client/src/components/sections/cooking.ts

```diff
@@ -2,21 +2,32 @@ import {htmlElement} from '@src/util/html';
 import {LightboxItem} from '../controls/lightbox';
 
 export interface DishInfo {
-  attributes: { name: string; image: string };
+  attributes: { name: string; image?: string };
   html: string;
 }
 
-export const CookingSection = (info: DishInfo): HTMLElement => htmlElement`
+export const CookingSection = (info: DishInfo): HTMLElement => {
+  const section: HTMLElement = htmlElement`
   <section class="cooking-section">
-    <div class="thumbnail">
-      ${LightboxItem({
-        // require returns the mangled URL to the image, via file-loader
-        src: require(`@res/img/dish/${info.attributes.image}`).default,
-        alt: info.attributes.name,
-      })}
-    </div>
     <div class="content">
       <h3>${info.attributes.name}</h3>
       ${info.html}
     </div>
   </section>`;
+
+  if (info.attributes.image) {
+    const src = require(`@res/img/dish/${info.attributes.image}?responsive&sizes[]=200,sizes[]=400,sizes[]=600`);
```

Comment:
nit: this is more than 80 characters. You can just make this 2 strings you truncate together to keep you under 80 chars.